src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,42 @@
+package lotto.domain;
+
+import static lotto.domain.Lotto.LOTTO_SIZE;
+import static lotto.domain.LottoNumber.MAX_NUMBER;
+import static lotto.domain.LottoNumber.MIN_NUMBER;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Lottos {
+    private final List<Lotto> lottos = new ArrayList<>();
+
+    public Lottos(PurchasePrice purchasePrice) {
+        for (int i = 0; i < purchasePrice.calculateLottoCount(); i++) {
+            lottos.add(generateLotto());
+        }
```

Comment:
stream()을 사용하시면 어떨까요?

```suggestion
       IntStream.range(0, purchasePrice.calculateLottoCount())
          .mapToObj(i -> generateLotto())
          .forEach(lottos::add);
```